### DISCUSSION TITLE
How do I report a problem just as a basic user, not as a developer who knows anything whatsoever about what anyone is talking about?

### DISCUSSION THREAD

**Discussion by julezyladee9** [Swordfish90/Lemuroid#401] [Q&A]
Look: I use F-Droid because I trust its apps and developers more than I do the ones on the Play Store. I love what they are doing and I appreciate the hell out of all the developers involved with F-Droid. I'm not trying to complain, but, I have a question that might not even be for the Lemuroid developers. I just don't know. I can't even easily figure out how to communicate with them here. No, it's not easy for someone who has no experience with this kind of site. IDK if I am supposed to report issues/ask questions here or in "Issues"-so, if I am wrong, please cut me some slack and realize that I am just trying to report an issue in order to make the app better for everyone, not just me.

I have only attempted playing one NES game (The Legend of Zelda), and am finding it impossible to use the controller when in the pause menu/area that shows the player's weapons and how much of the Triforce has been gathered. That means that I can't choose or change my secondary weapon, or otherwise move around the screen (if that is possible LOL I haven't played this game in a looooong time). Otherwise, the game works perfectly-which is why I am reporting it to the developers of Lemuroid instead of looking for a new copy of the game. Thank you for your help. I am really sorry if I'm wasting anyone's time. Please try not to be offended or irritated by my ignorance.

Gracias for a fantastic app. Peace

**Comment by rhizene** [comment]
Adding a comment here, would like to see what worked for others with psx GranTurismo being stuck just before the race
